src/Sylius/Bundle/CoreBundle/EventListener/MailerListener.php

```diff
@@ -32,6 +30,9 @@ class MailerListener
      */
     protected $emailSender;
 
+    /**
+     * @param SenderInterface $emailSender
+     */
```

Comment:
sorry but, i don't see the issue to document the constructor. if the attributes have already a comment